src/components/InstancePlugin/InstancePlugin.test.js

```diff
@@ -0,0 +1,25 @@
+import '../../../test/jest/__mock__';
+import { screen } from '@folio/jest-config-stripes/testing-library/react';
+import InstancePlugin from './InstancePlugin';
+import { renderWithIntl } from '../../../test/jest/helpers';
+
+const onSelectMock = jest.fn();
+
+describe('InstancePlugin', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should InstancePlugin', () => {
+    renderWithIntl(<InstancePlugin onSelectMock={onSelectMock} isDisabled={false} />);
```

Comment:
I think `onSelectMock` property is not expected.
Maybe you meant 
```suggestion
    renderWithIntl(<InstancePlugin onSelect={onSelectMock} isDisabled={false} />);
```

Reply:
Hi @artem-blazhko, fixed

cc @UladzislauKutarkin